libraries/YarpPlugins/CanBusControlboard/DeviceDriverImpl.cpp

```diff
@@ -23,6 +23,7 @@ bool roboticslab::CanBusControlboard::open(yarp::os::Searchable& config)
     yarp::os::Bottle minVels = config.findGroup("minVels").tail();  //-- e.g. 0
     yarp::os::Bottle refAccelerations = config.findGroup("refAccelerations").tail();  //-- e.g. 0.575437
     yarp::os::Bottle refSpeeds = config.findGroup("refSpeeds").tail();  //-- e.g. 737.2798
+    yarp::os::Bottle encoderPulses = config.findGroup("encoderPulses").tail();  //-- e.g. 4096 (4 * 1024)
```

Comment:
`encoderPulsess` (note the double 's'). Compare with line 78 on this same file (TechnosoftIpos reads a `encoderPulses` - one 's').